### ISSUE TITLE
A new local covariance matrix estimation for the classification of gene expression profiles in high dimensional RNA-Seq data

### ISSUE THREAD

**Issue by cmclean5** [cmclean5/PublicHealthModels#31]
## TL;DR

Recent developments in the next-generation sequencing based on RNA-sequencing (RNA-Seq) allow researchers to measure the expression levels of thousands of genes for multiple samples simultaneously. Paper propose a new approach based on local dependence function to estimate the covariance matrix for RNA-seq data.

### Paper Link

https://www.sciencedirect.com/science/article/pii/S0957417420309295?via=ihub

### Author/Institution

 Necla Kochan (Izmir Biomedicine and Genome Center, Izmir, Turkey)

## Cite

Necla Kochan, G. Yazgı Tütüncü, Göknur Giner,
A new local covariance matrix estimation for the classification of gene expression profiles in high dimensional RNA-Seq data,
Expert Systems with Applications,
Volume 167,
2021,
114200,
ISSN 0957-4174,
https://doi.org/10.1016/j.eswa.2020.114200.
(https://www.sciencedirect.com/science/article/pii/S0957417420309295)
